go/database/mpt/verification.go

```diff
@@ -41,15 +42,11 @@ func (NilVerificationObserver) StartVerification()        {}
 func (NilVerificationObserver) Progress(msg string)       {}
 func (NilVerificationObserver) EndVerification(res error) {}
 
-// VerifyFileForest runs list of validation checks on the forest stored in the given
-// directory. These checks include:
-//   - all required files are present and can be read
-//   - all referenced nodes are present
-//   - all hashes are consistent
-func VerifyFileForest(directory string, config MptConfig, roots []Root, observer VerificationObserver) (res error) {
+func VerifyMptState(directory string, config MptConfig, roots []Root, observer VerificationObserver) (res error) {
```

Comment:
Please document, as this function seems to be intended for package-external use. If not, make it private.